### ISSUE TITLE
Pretrained Model (.Pth)

### ISSUE THREAD

**Issue by luongthecong123** [svip-lab/RGBD-Counting#4]
I wonder if you could provide the pretrained weight (.pth) to this project ? Thank you.

**Comment by TengfeiHou** [comment]
+1

**Comment by ashesh-0** [comment]
Yeah, it would be great to have pre-trained weights ! (Readme docs at places assumes that checkpoints have been downloaded)

**Comment by 1691077362** [comment]
@ @ashesh-0 +1
